build.gradle

```diff
@@ -40,8 +40,8 @@ group = "org.jabref"
 version = project.findProperty('projVersion') ?: '100.0.0'
 
 java {
-    sourceCompatibility = JavaVersion.VERSION_18
-    targetCompatibility = JavaVersion.VERSION_18
+    sourceCompatibility = JavaVersion.VERSION_19
```

Comment:
I would leave this with 18, so that oeople who stil use 18 can use it as well

Reply:
Since we use jlink to compile the jdk with JabRef for distribution it does not matter for the end-user.
For the developer the sourceCompatibility is just a safetynet for not using newer features. The targetCompatibility is the JDK we are using. But we want to be able to use the latest JDK with all the latest security patches. Alternative would be to switch back to 17 LTS, but not to stay stuck on 18.
https://www.baeldung.com/gradle-sourcecompatiblity-vs-targetcompatibility